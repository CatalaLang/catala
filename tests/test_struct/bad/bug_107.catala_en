## https://github.com/CatalaLang/catala/issues/107

```catala
declaration structure S:
  data x content integer
  data y content integer

declaration structure S:
  data x content integer
  data y content integer

declaration scope A:
  context a content S

scope A:
  definition a equals S { --x : 0 -- y : 1 }
```
