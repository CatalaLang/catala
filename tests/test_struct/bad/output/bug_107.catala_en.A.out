[ERROR] struct name "S" already defined
[ERROR] 
[ERROR] First definition:
[ERROR]  --> test_struct/bad/bug_107.catala_en
[ERROR]   | 
[ERROR] 4 | declaration structure S:
[ERROR]   |                       ^
[ERROR]   + https://github.com/CatalaLang/catala/issues/107
[ERROR] 
[ERROR] Second definition:
[ERROR]  --> test_struct/bad/bug_107.catala_en
[ERROR]   | 
[ERROR] 8 | declaration structure S:
[ERROR]   |                       ^
[ERROR]   + https://github.com/CatalaLang/catala/issues/107
